### PR TITLE
add logging fuc

### PR DESCRIPTION
**Pull request by killpanda**

### CODE REVIEW

**Comment by GuoJing** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -65,6 +65,16 @@ def hello():
     return 'hello world'
 ```
 
+add key, value for your tracing record, like:
+
+```python
+
+zipkin.logging(id=1, user_id=2)
```
Comment (adriancole) [inline README.md]: I think this isn't what logging usually means. more typically it is tagging, right?

Comment (GuoJing) [inline README.md]: @adriancole hmmm... logging maybe not the name. I think `logging` here means send the span with annotations.

**Comment by adriancole** [comment]
just noting that often logging implies binding an event to a timestamp.
binary annotations are tags that don't have timestamps. I'm not sure if
I've seen libraries who do the same as what's being done here logging
before.

**Comment by killpanda** [comment]
@adriancole thanks, we will rename this method  :)

**Comment by GuoJing** [comment]
I agree logging should change to a better name.

**Pull request merged**